sdk/iothub/azure-iot-deviceprovisioningservice/azure/iot/deviceprovisioningservice/auth.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
```

Comment:
This module can be marked as private, `_auth.py`, useful public attributes like `generate_sas_token` can be exposed in deviceprovisioningservice/\_\_init__.py  \_\_all__